### PR TITLE
add a communications section to How We Work

### PR DESCRIPTION
**Pull request by metadave**
cc @bensternthal @pmac @jgmize @jwhitlock

### CODE REVIEW

**Comment by bensternthal** [comment]
@metadave Are you thinking this is how we use irc/slack or more along the lines of escalation path in case of a problem... or both!

**Comment by metadave** [comment]
@bensternthal it's more for upcoming work that we need to communicate, I pushed some examples to clarify.

File: docs/how_we_work.md [hunk 1]
```diff
@@ -135,3 +135,43 @@ The task is complete, it is _live_ and no additional work is required.
...
+- Include the following:
+  - future work that may impact other projects or teams. 
+  - mention non-SRE's that have contributed to SRE-related tasks
+
+## Work that will impact other teams and projects
+
+When SRE work includes activities that may impact another team or project, communicate these changes so the appropriate engineers have time to prepare:
+
+- create an issue in the appropriate Github repo stating the change and impact. Include a link back to the SRE issue in the [infra repo](https://github.com/mozmeao/infra).	 
+	- be courteous: ask the team how much notice they need to prepare.
```
Comment (pmac) [inline docs/how_we_work.md]: Seem to be mixing tab indents with space indents in the file. Might be odd for rendering.

File: docs/how_we_work.md [hunk 2]
```diff
@@ -135,3 +135,43 @@ The task is complete, it is _live_ and no additional work is required.
...
+
+- deploying applications to new clusters
+	- this may require changes to Jenkins deployments
+	- a team member that manages the app may need to run some tests against the app in a new cluster
+- deommissioning clusters
+	- this may require changes to Jenkins deployments
+	- this may affect performance if teams are gathering specific metrics
+- changing app infrastructure
+	- switching between Deis 1/K8s/S3 hosting/etc
+
```
Comment (pmac) [inline docs/how_we_work.md]: Some extra blank lines at the end.

**Comment by pmac** [comment]
This looks fantastic to me Dave! Appreciate your work on this! 💯

File: docs/how_we_work.md (hunk 1 above)
Comment (metadave) [inline docs/how_we_work.md]: I'm using MacDown, and the settings somehow reverted. Thanks for the heads up, fixed!

**Pull request merged**